tasks/symlinks.yml

```diff
@@ -0,0 +1,37 @@
+- name: get all subdirectories
+  find:
+    paths: "{{service_item}}/live"
+    recurse: no
+    file_type: directory
+  register: service_subdirs
+  become: yes
+  become_user: root
+
+- name: get all hosts
+  set_fact:
+    service_hosts: "{{service_subdirs.files | map(attribute='path') | map('regex_replace', service_item + '/live/') | list}}"
+
```

Comment:
Not needed, see below.

Reply:
I definitely need to extract the hostname from the directory path, in order to name the symlink I'm creating.

Reply:
But the `file` command is going to be

```
- name: Create symbolic links
  file:
    src: '{{item.path}}/fullchain_privkey.pem'
    dest: '{{haproxy_certs_directory}}/{{ item.path | basename}}.pem
    state: link
    follow: no
  loop: '{{service_subdirs.files}}'
```